FreeRTOS-Plus/Demo/Common/Logging/windows/Logging_WinSim.c

```diff
@@ -403,16 +403,15 @@ void vLoggingPrintf( const char * pcFormat,
             if( xLength2 >= ( xLength + sizeof( xLength ) ) )
             {
                 /* First write in the length of the data, then write in the data
-                 * itself.  Raising the thread priority is used as a critical section
-                 * as there are potentially multiple writers.  The stream buffer is
-                 * only thread safe when there is a single writer (likewise for
-                 * reading from the buffer). */
-                xCurrentTask = GetCurrentThread();
-                iOriginalPriority = GetThreadPriority( xCurrentTask );
-                SetThreadPriority( GetCurrentThread(), THREAD_PRIORITY_TIME_CRITICAL );
```

Comment:
A naive question: Why wasn't this able to achieve the same thing as `taskENTER_CRITICAL()`?